configs/config.cfg

```diff
@@ -31,27 +31,24 @@ scorer = {"@scorers":"spacy.textcat_multilabel_scorer.v2"}
 threshold = 0.5
 
 [components.textcat_multilabel.model]
-@architectures = "spacy.TextCatCNN.v1"
+@architectures = "spacy.TextCatReduce.v1"
```

Comment:
What is the motivation for this change from CNN to Reduce?

Reply:
Från SpaCys dokumentation:
> TextCatCNN has been replaced by the more general [TextCatReduce](https://spacy.io/api/architectures#TextCatReduce)